src/ui/mails/Mails.js

```diff
@@ -47,65 +59,86 @@ class Mails extends React.Component {
+            this.state.currentEmail = null;
+            this.forceUpdate();
+        }
+
         const data = this.state.mails;
+        const currentEmail = this.state.currentEmail;
 
         return <div>
             <div style={{ textAlign: 'right' }}>
+                <div style={{ display: 'inline-block', marginRight: '5px' }}>
+                    <label>Filter on to:&nbsp;</label>
+                    <input type="search" onChange={this.filterOnEmailTo} />
+                </div>
                 <a href='#' onClick={this.refresh}>Refresh</a>
             </div>
 
+            {currentEmail ? <PopUp currentEmail={currentEmail} hide={hide} /> : null }
+
             <ReactTable
                 style={{ marginTop: '12px' }}
                 data={data}
+                showPageSizeOptions={false}
                 columns={[{
                     Header: 'Mails',
                     columns: [
                         {
                             Header: 'datetime',
                             id: 'datetime',
+                            headerStyle: {textAlign: 'left'},
+                            width: 220,
                             accessor: mail => mail.datetime
                         },
                         {
                             Header: 'from',
                             id: 'from',
+                            headerStyle: {textAlign: 'left'},
+                            width: 220,
                             accessor: mail => (mail.from ? mail.from.email : '')
                         },
                         {
                             Header: 'subject',
                             id: 'subject',
+                            headerStyle: {textAlign: 'left'},
                             style: { 'whiteSpace': 'unset' },
+                            minWidth: 200,
                             accessor: mail => mail.subject
                         },
                         {
                             Header: 'to',
                             id: 'to',
+                            headerStyle: {textAlign: 'left'},
                             accessor: mail => mail.personalizations,
                             Cell: cellData => (
                                 cellData.value
                                     .filter(value => !!value.to)
                                     .map(value => value.to)
                                     .map((tos, index) => (
-                                        <ul key={index}>
-                                            {tos.map((to, index) =>
-                                                (<li key={index}>{to.email}</li>))}
-                                        </ul>)
+                                        <span key={index}>
+                                            {tos.map((to, index) => 
+                                                (<span key={index}>{to.email}</span> ))}
+                                        </span>)
```

Comment:
Multiple email addresses should be visualized as list, the use of `span` does not match this requirement.

Reply:
👍